### PR TITLE
refactor: Use the flexible camera manager in 360 images

### PR DESCRIPTION
**Pull request by nilscognite**
#### Type of change

Refactor 

#### Jira ticket :blue_book:

No Jira ticket

https://cognitedata.atlassian.net/browse/

## Description :pencil:

This is a refactor to get rid of the Stationary Camera manager, since it behaved a little different then the FlexibleCamera Manager. Now the Use arrows and AxisGizmo is working inside the 360 image.

## How has this been tested? :mag:

Use a model with 3D images. Jump in and out of the images. Zoom and pan. Use arrows and AxisGizmo to navigate.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: viewer/packages/api/src/api-helpers/Image360ApiHelper.ts [hunk 1]
```diff
@@ -70,9 +76,9 @@ export class Image360ApiHelper {
   );
 
   private readonly _activeCameraManager: ProxyCameraManager;
-  private readonly _image360Navigation: StationaryCameraManager;
+  private readonly _stationaryCameraManager: StationaryCameraManager | undefined = undefined;
```
Comment (pramodcog) [inline viewer/packages/api/src/api-helpers/Image360ApiHelper.ts:79]: In TypeScript, we do not need to explicitly assign `undefined` value while creating the variable, it automatically assigns `undefined` value when created!

File: viewer/packages/api/src/api-helpers/Image360ApiHelper.ts [hunk 2]
```diff
@@ -33,9 +33,15 @@ import {
...
-import { CameraManager, ProxyCameraManager, StationaryCameraManager } from '@reveal/camera-manager';
+import {
+  CameraManager,
+  FlexibleCameraManager,
+  ProxyCameraManager,
+  StationaryCameraManager
+} from '@reveal/camera-manager';
 import { MetricsLogger } from '@reveal/metrics';
 import debounce from 'lodash/debounce';
+import { moveCameraPositionTo, tweenCameraToDefaultFov } from '@reveal/camera-manager/src/Flexible/moveCamera';
```
Comment (pramodcog) [inline viewer/packages/api/src/api-helpers/Image360ApiHelper.ts:44]: `moveCameraPositionTo` and `tweenCameraToDefaultFov` should be exported within `camera-manager` package and imported as `from '@reveaal/camera-manager`

File: viewer/packages/api/src/api-helpers/Image360ApiHelper.ts (hunk 1 above)
Comment (nilscognite) [reply viewer/packages/api/src/api-helpers/Image360ApiHelper.ts]: Fixed

File: viewer/packages/api/src/api-helpers/Image360ApiHelper.ts (hunk 2 above)
Comment (nilscognite) [reply viewer/packages/api/src/api-helpers/Image360ApiHelper.ts]: I move these functions in the end of the file instead

File: viewer/packages/api/src/api-helpers/Image360ApiHelper.ts [hunk 3]
```diff
@@ -529,3 +565,61 @@ export class Image360ApiHelper {
     this.exit360Image();
   }
 }
+
+//================================================
+// STATIC METHODS: Tweens for FlexibleCameraManager
+//================================================
+
+function moveCameraPositionTo(manager: FlexibleCameraManager, position: Vector3, duration: number): void {
```
Comment (pramodcog) [inline viewer/packages/api/src/api-helpers/Image360ApiHelper.ts:573]: I would prefer to have proper name `manager` ---> `cameraManager`!

File: viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts [hunk 4]
```diff
@@ -148,6 +151,10 @@ export class FlexibleControls extends EventDispatcher<FlexibleControlsEvent> {
     return this._camera;
   }
 
+  get domElement(): HTMLElement {
```
Comment (pramodcog) [inline viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts:154]: Why controls should expose `DOM Element`, isn't `viewer._domElement` and this same?

**Review by pramodcog — APPROVED** [review]
Looks good! minor comment

File: viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts (hunk 4 above)
Comment (nilscognite) [reply viewer/packages/camera-manager/src/Flexible/FlexibleControls.ts:154]: I don' have the viewer here

**Pull request merged**